src/routes/api/tree.ts

```diff
@@ -122,11 +127,11 @@ function getNotesAndBranchesAndAttributes(noteIds) {
     };
 }
 
-function getTree(req) {
-    const subTreeNoteId = req.query.subTreeNoteId || 'root';
-    const collectedNoteIds = new Set([subTreeNoteId]);
+function getTree(req: Request) {
+    const subTreeNoteId = req.query.subTreeNoteId === "string" ? req.query.subTreeNoteId : "" || 'root';
```

Comment:
I think you mean `const subTreeNoteId = *typeof* req.query.subTreeNoteId === "string" ? req.query.subTreeNoteId : "" || 'root';`

Reply:
@alexpietsch , this is a pretty good find on your end. Thank you! :)
Most likely this would have caused quite a bit of a headache when trying to identify what caused this regression.